package.json

```diff
@@ -6,7 +6,7 @@
     "start": "node src/start.js",
     "lint": "eslint . --ext .js",
     "test": "jest -c jest.config.js",
-    "test:unit": "jest -c jest.config.unit.js",
+    "test:unit": "jest --detectOpenHandles -c jest.config.unit.js",
```

Comment:
This is probably not something you'd want in production code, but since the number of tests is not very high, it might not be an issue.

Reply:
I'll remove it. Good call out.

Reply:
Nice catch